### PR TITLE
Add NoArgs to all commands 

### PR DESCRIPTION
**Pull request by asddongmen**
### What problem does this PR solve? 
#2303 

### What is changed and how it works?
add noArgs to all commands 

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

Side effects

 - Breaking backward compatibility

Related changes

 - Need to cherry-pick to the release branch

### Release note 

```release-note
None
```

### CODE REVIEW

File: tests/cdc/run.sh [hunk 1]
```diff
@@ -18,6 +18,9 @@ function prepare() {
 
     # record tso before we create tables to skip the system table DDLs
     start_ts=$(run_cdc_cli tso query --pd=http://$UP_PD_HOST_1:$UP_PD_PORT_1)
```
Comment (hi-rustin) [inline tests/cdc/run.sh:20]: Maybe we need to extract it into a function? It looks like it repeats many times. Maybe we can just awk it after it finishes running in run_cdc_cli?

Comment (asddongmen) [reply tests/cdc/run.sh:20]: ok，good idea.

**Comment by hi-rustin** [comment]
block by https://github.com/spf13/cobra/issues/1480

File: tests/_utils/query_tso [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+# parameter 1: pd host
+# parameter 2: pd port
+
+set -eux
+
+pd_host=${1}
+pd_port=${2}
+
+tso=$(run_cdc_cli tso query --pd=http://${pd_host}:${pd_port})
+echo $tso | awk -F " " '{print $1}'
```
Comment (hi-rustin) [inline tests/_utils/query_tso:12]: Maybe we need to add some comments here?

**Review by 3AceShowHand — APPROVED** [review]

File: tests/_utils/query_tso [hunk 3]
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
```
Comment (hi-rustin) [inline tests/_utils/query_tso:1]: Maybe `run_cdc_cli_tso_query` is better?

Comment (asddongmen) [reply tests/_utils/query_tso:1]: good

**Review by hi-rustin — APPROVED** [review]

**Comment by amyangfei** [comment]
https://ci2.pingcap.net/blue/organizations/jenkins/cdc_ghpr_integration_test/detail/cdc_ghpr_integration_test/4531/pipeline/111
strange error:

```
[2021-09-23T11:10:17.425Z] [WARN] --sort-dir is deprecated in changefeed settings. Please use `cdc server --data-dir` to start the cdc server if possible, sort-dir will be set automatically. The --sort-dir here will be no-op
[2021-09-23T11:10:17.425Z] [WARN] --sort-dir is deprecated in changefeed settings. Please use `cdc server --data-dir` to start the cdc server if possible, sort-dir will be set automatically. The --sort-dir here will be no-op
[2021-09-23T11:10:17.425Z] Replicate lag (34h12m5.112s) is larger than 1 days, large data may cause OOM, confirm to continue at your own risk [Y/N]
[2021-09-23T11:10:17.425Z] Replicate lag (34h12m5.112s) is larger than 1 days, large data may cause OOM, confirm to continue at your own risk [Y/N]
[2021-09-23T11:10:17.425Z] Error: unknown command "\u3000-c=" for "create"
[2021-09-23T11:10:17.425Z] Usage:
[2021-09-23T11:10:17.425Z]   create [flags]
[2021-09-23T11:10:17.425Z] 
[2021-09-23T11:10:17.425Z] Flags:
[2021-09-23T11:10:17.425Z]   -c, --changefeed-id string              Replication task (changefeed) ID
[2021-09-23T11:10:17.425Z]       --config string                     Path of the configuration file
[2021-09-23T11:10:17.426Z]       --cyclic-filter-replica-ids uints   (Experimental) Cyclic replication filter replica ID of changefeed (default [])
[2021-09-23T11:10:17.426Z]       --cyclic-replica-id uint            (Experimental) Cyclic replication replica ID of changefeed
[2021-09-23T11:10:17.426Z]       --cyclic-sync-ddl                   (Experimental) Cyclic replication sync DDL of changefeed (default true)
[2021-09-23T11:10:17.426Z]       --disable-gc-check                  Disable GC safe point check
[2021-09-23T11:10:17.426Z]   -h, --help                              help for create
[2021-09-23T11:10:17.426Z]       --no-confirm                        Don't ask user whether to ignore ineligible table
[2021-09-23T11:10:17.426Z]       --opts key=value                    Extra options, in the key=value format
[2021-09-23T11:10:17.426Z]       --sink-uri string                   sink uri
[2021-09-23T11:10:17.426Z]       --sort-engine string                sort engine used for data sort (default "unified")
[2021-09-23T11:10:17.426Z]       --start-ts uint                     Start ts of changefeed
[2021-09-23T11:10:17.426Z]       --sync-interval duration            (Experimental) Set the interval for syncpoint in replication(default 10min) (default 10m0s)
[2021-09-23T11:10:17.426Z]       --sync-point                        (Experimental) Set and Record syncpoint in replication(default off)
[2021-09-23T11:10:17.426Z]       --target-ts uint                    Target ts of changefeed
[2021-09-23T11:10:17.426Z]       --tz string                         timezone used when checking sink uri (changefeed timezone is determined by cdc server) (default "SYSTEM")
[2021-09-23T11:10:17.426Z] 
[2021-09-23T11:10:17.426Z] Error: unknown command "aa" for "create"
[2021-09-23T11:10:17.426Z] Usage:
[2021-09-23T11:10:17.426Z]   create [flags]
[2021-09-23T11:10:17.426Z] 
[2021-09-23T11:10:17.426Z] Flags:
[2021-09-23T11:10:17.426Z]   -c, --changefeed-id string              Replication task (changefeed) ID
[2021-09-23T11:10:17.426Z]       --config string                     Path of the configuration file
[2021-09-23T11:10:17.426Z]       --cyclic-filter-replica-ids uints   (Experimental) Cyclic replication filter replica ID of changefeed (default [])
[2021-09-23T11:10:17.426Z]       --cyclic-replica-id uint            (Experimental) Cyclic replication replica ID of changefeed
[2021-09-23T11:10:17.426Z]       --cyclic-sync-ddl                   (Experimental) Cyclic replication sync DDL of changefeed (default true)
[2021-09-23T11:10:17.426Z]       --disable-gc-check                  Disable GC safe point check
[2021-09-23T11:10:17.426Z]   -h, --help                              help for create
[2021-09-23T11:10:17.426Z]       --no-confirm                        Don't ask user whether to ignore ineligible table
[2021-09-23T11:10:17.426Z]       --opts key=value                    Extra options, in the key=value format
[2021-09-23T11:10:17.426Z]       --sink-uri string                   sink uri
[2021-09-23T11:10:17.426Z]       --sort-engine string                sort engine used for data sort (default "unified")
[2021-09-23T11:10:17.426Z]       --start-ts uint                     Start ts of changefeed
[2021-09-23T11:10:17.426Z]       --sync-interval duration            (Experimental) Set the interval for syncpoint in replication(default 10min) (default 10m0s)
[2021-09-23T11:10:17.426Z]       --sync-point                        (Experimental) Set and Record syncpoint in replication(default off)
[2021-09-23T11:10:17.426Z]       --target-ts uint                    Target ts of changefeed
[2021-09-23T11:10:17.426Z]       --tz string                         timezone used when checking sink uri (changefeed timezone is determined by cdc server) (default "SYSTEM")
[2021-09-23T11:10:17.426Z] 
[2021-09-23T11:10:17.426Z] Error: unknown command "国--start-ts=0" for "create"
[2021-09-23T11:10:17.426Z] Usage:
[2021-09-23T11:10:17.426Z]   create [flags]
[2021-09-23T11:10:17.426Z] 
[2021-09-23T11:10:17.426Z] Flags:
[2021-09-23T11:10:17.426Z]   -c, --changefeed-id string              Replication task (changefeed) ID
[2021-09-23T11:10:17.426Z]       --config string                     Path of the configuration file
[2021-09-23T11:10:17.426Z]       --cyclic-filter-replica-ids uints   (Experimental) Cyclic replication filter replica ID of changefeed (default [])
[2021-09-23T11:10:17.426Z]       --cyclic-replica-id uint            (Experimental) Cyclic replication replica ID of changefeed
[2021-09-23T11:10:17.426Z]       --cyclic-sync-ddl                   (Experimental) Cyclic replication sync DDL of changefeed (default true)
[2021-09-23T11:10:17.426Z]       --disable-gc-check                  Disable GC safe point check
[2021-09-23T11:10:17.426Z]   -h, --help                              help for create
[2021-09-23T11:10:17.426Z]       --no-confirm                        Don't ask user whether to ignore ineligible table
[2021-09-23T11:10:17.426Z]       --opts key=value                    Extra options, in the key=value format
[2021-09-23T11:10:17.426Z]       --sink-uri string                   sink uri
[2021-09-23T11:10:17.426Z]       --sort-engine string                sort engine used for data sort (default "unified")
[2021-09-23T11:10:17.426Z]       --start-ts uint                     Start ts of changefeed
[2021-09-23T11:10:17.426Z]       --sync-interval duration            (Experimental) Set the interval for syncpoint in replication(default 10min) (default 10m0s)
[2021-09-23T11:10:17.426Z]       --sync-point                        (Experimental) Set and Record syncpoint in replication(default off)
[2021-09-23T11:10:17.426Z]       --target-ts uint                    Target ts of changefeed
[2021-09-23T11:10:17.426Z]       --tz string                         timezone used when checking sink uri (changefeed timezone is determined by cdc server) (default "SYSTEM")
[2021-09-23T11:10:17.426Z] 
[2021-09-23T11:10:17.426Z] [WARN] TiCDC server data-dir is not set. Please use `cdc server --data-dir` to start the cdc server if possible.
[2021-09-23T11:10:17.426Z] [WARN] TiCDC server data-dir is not set. Please use `cdc server --data-dir` to start the cdc server if possible.
[2021-09-23T11:10:17.426Z] [WARN] TiCDC server data-dir is not set. Please use `cdc server --data-dir` to start the cdc server if possible.
[2021-09-23T11:10:17.426Z] [WARN] --sort-dir is deprecated in server settings. sort-dir will be set to `{data-dir}/tmp/sorter`. The sort-dir here will be no-op
[2021-09-23T11:10:17.426Z] [WARN] --sort-dir is deprecated in server settings. sort-dir will be set to `{data-dir}/tmp/sorter`. The sort-dir here will be no-op
[2021-09-23T11:10:17.426Z] [WARN] --sort-dir is deprecated in server settings. sort-dir will be set to `{data-dir}/tmp/sorter`. The sort-dir here will be no-op
[2021-09-23T11:10:17.426Z] 
[2021-09-23T11:10:17.426Z] ----------------------------------------------------------------------
[2021-09-23T11:10:17.426Z] FAIL: util_test.go:60: utilsSuite.TestVerifyPdEndpoint
[2021-09-23T11:10:17.426Z] 
[2021-09-23T11:10:17.426Z] util_test.go:68:
[2021-09-23T11:10:17.426Z]     c.Assert(verifyPdEndpoint(url, false), check.ErrorMatches, ".*invalid control character in URL.*")
[2021-09-23T11:10:17.426Z] ... error string = "" +
[2021-09-23T11:10:17.426Z] ...     "parse PD endpoint: parse \n" +
[2021-09-23T11:10:17.426Z] ...     " hi: net/url: invalid control character in URL"
[2021-09-23T11:10:17.426Z] ... regex string = ".*invalid control character in URL.*"
[2021-09-23T11:10:17.426Z] 
[2021-09-23T11:10:17.426Z] OOPS: 13 passed, 1 FAILED
[2021-09-23T11:10:17.426Z] --- FAIL: TestSuite (0.06s)
[2021-09-23T11:10:17.426Z] FAIL
[2021-09-23T11:10:17.426Z] coverage: 33.6% of statements
[2021-09-23T11:10:17.426Z] FAIL	github.com/pingcap/ticdc/cmd	0.216s
```

**Comment by asddongmen** [comment]
> https://ci2.pingcap.net/blue/organizations/jenkins/cdc_ghpr_integration_test/detail/cdc_ghpr_integration_test/4531/pipeline/111
> strange error:
> 
> ```
> [2021-09-23T11:10:17.425Z] [WARN] --sort-dir is deprecated in changefeed settings. Please use `cdc server --data-dir` to start the cdc server if possible, sort-dir will be set automatically. The --sort-dir here will be no-op
> [2021-09-23T11:10:17.425Z] [WARN] --sort-dir is deprecated in changefeed settings. Please use `cdc server --data-dir` to start the cdc server if possible, sort-dir will be set automatically. The --sort-dir here will be no-op
> [2021-09-23T11:10:17.425Z] Replicate lag (34h12m5.112s) is larger than 1 days, large data may cause OOM, confirm to continue at your own risk [Y/N]
> [2021-09-23T11:10:17.425Z] Replicate lag (34h12m5.112s) is larger than 1 days, large data may cause OOM, confirm to continue at your own risk [Y/N]
> [2021-09-23T11:10:17.425Z] Error: unknown command "\u3000-c=" for "create"
> [2021-09-23T11:10:17.425Z] Usage:
> [2021-09-23T11:10:17.425Z]   create [flags]
> [2021-09-23T11:10:17.425Z] 
> [2021-09-23T11:10:17.425Z] Flags:
> [2021-09-23T11:10:17.425Z]   -c, --changefeed-id string              Replication task (changefeed) ID
> [2021-09-23T11:10:17.425Z]       --config string                     Path of the configuration file
> [2021-09-23T11:10:17.426Z]       --cyclic-filter-replica-ids uints   (Experimental) Cyclic replication filter replica ID of changefeed (default [])
> [2021-09-23T11:10:17.426Z]       --cyclic-replica-id uint            (Experimental) Cyclic replication replica ID of changefeed
> [2021-09-23T11:10:17.426Z]       --cyclic-sync-ddl                   (Experimental) Cyclic replication sync DDL of changefeed (default true)
> [2021-09-23T11:10:17.426Z]       --disable-gc-check                  Disable GC safe point check
> [2021-09-23T11:10:17.426Z]   -h, --help                              help for create
> [2021-09-23T11:10:17.426Z]       --no-confirm                        Don't ask user whether to ignore ineligible table
> [2021-09-23T11:10:17.426Z]       --opts key=value                    Extra options, in the key=value format
> [2021-09-23T11:10:17.426Z]       --sink-uri string                   sink uri
> [2021-09-23T11:10:17.426Z]       --sort-engine string                sort engine used for data sort (default "unified")
> [2021-09-23T11:10:17.426Z]       --start-ts uint                     Start ts of changefeed
> [2021-09-23T11:10:17.426Z]       --sync-interval duration            (Experimental) Set the interval for syncpoint in replication(default 10min) (default 10m0s)
> [2021-09-23T11:10:17.426Z]       --sync-point                        (Experimental) Set and Record syncpoint in replication(default off)
> [2021-09-23T11:10:17.426Z]       --target-ts uint                    Target ts of changefeed
> [2021-09-23T11:10:17.426Z]       --tz string                         timezone used when checking sink uri (changefeed timezone is determined by cdc server) (default "SYSTEM")
> [2021-09-23T11:10:17.426Z] 
> [2021-09-23T11:10:17.426Z] Error: unknown command "aa" for "create"
> [2021-09-23T11:10:17.426Z] Usage:
> [2021-09-23T11:10:17.426Z]   create [flags]
> [2021-09-23T11:10:17.426Z] 
> [2021-09-23T11:10:17.426Z] Flags:
> [2021-09-23T11:10:17.426Z]   -c, --changefeed-id string              Replication task (changefeed) ID
> [2021-09-23T11:10:17.426Z]       --config string                     Path of the configuration file
> [2021-09-23T11:10:17.426Z]       --cyclic-filter-replica-ids uints   (Experimental) Cyclic replication filter replica ID of changefeed (default [])
> [2021-09-23T11:10:17.426Z]       --cyclic-replica-id uint            (Experimental) Cyclic replication replica ID of changefeed
> [2021-09-23T11:10:17.426Z]       --cyclic-sync-ddl                   (Experimental) Cyclic replication sync DDL of changefeed (default true)
> [2021-09-23T11:10:17.426Z]       --disable-gc-check                  Disable GC safe point check
> [2021-09-23T11:10:17.426Z]   -h, --help                              help for create
> [2021-09-23T11:10:17.426Z]       --no-confirm                        Don't ask user whether to ignore ineligible table
> [2021-09-23T11:10:17.426Z]       --opts key=value                    Extra options, in the key=value format
> [2021-09-23T11:10:17.426Z]       --sink-uri string                   sink uri
> [2021-09-23T11:10:17.426Z]       --sort-engine string                sort engine used for data sort (default "unified")
> [2021-09-23T11:10:17.426Z]       --start-ts uint                     Start ts of changefeed
> [2021-09-23T11:10:17.426Z]       --sync-interval duration            (Experimental) Set the interval for syncpoint in replication(default 10min) (default 10m0s)
> [2021-09-23T11:10:17.426Z]       --sync-point                        (Experimental) Set and Record syncpoint in replication(default off)
> [2021-09-23T11:10:17.426Z]       --target-ts uint                    Target ts of changefeed
> [2021-09-23T11:10:17.426Z]       --tz string                         timezone used when checking sink uri (changefeed timezone is determined by cdc server) (default "SYSTEM")
> [2021-09-23T11:10:17.426Z] 
> [2021-09-23T11:10:17.426Z] Error: unknown command "国--start-ts=0" for "create"
> [2021-09-23T11:10:17.426Z] Usage:
> [2021-09-23T11:10:17.426Z]   create [flags]
> [2021-09-23T11:10:17.426Z] 
> [2021-09-23T11:10:17.426Z] Flags:
> [2021-09-23T11:10:17.426Z]   -c, --changefeed-id string              Replication task (changefeed) ID
> [2021-09-23T11:10:17.426Z]       --config string                     Path of the configuration file
> [2021-09-23T11:10:17.426Z]       --cyclic-filter-replica-ids uints   (Experimental) Cyclic replication filter replica ID of changefeed (default [])
> [2021-09-23T11:10:17.426Z]       --cyclic-replica-id uint            (Experimental) Cyclic replication replica ID of changefeed
> [2021-09-23T11:10:17.426Z]       --cyclic-sync-ddl                   (Experimental) Cyclic replication sync DDL of changefeed (default true)
> [2021-09-23T11:10:17.426Z]       --disable-gc-check                  Disable GC safe point check
> [2021-09-23T11:10:17.426Z]   -h, --help                              help for create
> [2021-09-23T11:10:17.426Z]       --no-confirm                        Don't ask user whether to ignore ineligible table
> [2021-09-23T11:10:17.426Z]       --opts key=value                    Extra options, in the key=value format
> [2021-09-23T11:10:17.426Z]       --sink-uri string                   sink uri
> [2021-09-23T11:10:17.426Z]       --sort-engine string                sort engine used for data sort (default "unified")
> [2021-09-23T11:10:17.426Z]       --start-ts uint                     Start ts of changefeed
> [2021-09-23T11:10:17.426Z]       --sync-interval duration            (Experimental) Set the interval for syncpoint in replication(default 10min) (default 10m0s)
> [2021-09-23T11:10:17.426Z]       --sync-point                        (Experimental) Set and Record syncpoint in replication(default off)
> [2021-09-23T11:10:17.426Z]       --target-ts uint                    Target ts of changefeed
> [2021-09-23T11:10:17.426Z]       --tz string                         timezone used when checking sink uri (changefeed timezone is determined by cdc server) (default "SYSTEM")
> [2021-09-23T11:10:17.426Z] 
> [2021-09-23T11:10:17.426Z] [WARN] TiCDC server data-dir is not set. Please use `cdc server --data-dir` to start the cdc server if possible.
> [2021-09-23T11:10:17.426Z] [WARN] TiCDC server data-dir is not set. Please use `cdc server --data-dir` to start the cdc server if possible.
> [2021-09-23T11:10:17.426Z] [WARN] TiCDC server data-dir is not set. Please use `cdc server --data-dir` to start the cdc server if possible.
> [2021-09-23T11:10:17.426Z] [WARN] --sort-dir is deprecated in server settings. sort-dir will be set to `{data-dir}/tmp/sorter`. The sort-dir here will be no-op
> [2021-09-23T11:10:17.426Z] [WARN] --sort-dir is deprecated in server settings. sort-dir will be set to `{data-dir}/tmp/sorter`. The sort-dir here will be no-op
> [2021-09-23T11:10:17.426Z] [WARN] --sort-dir is deprecated in server settings. sort-dir will be set to `{data-dir}/tmp/sorter`. The sort-dir here will be no-op
> [2021-09-23T11:10:17.426Z] 
> [2021-09-23T11:10:17.426Z] ----------------------------------------------------------------------
> [2021-09-23T11:10:17.426Z] FAIL: util_test.go:60: utilsSuite.TestVerifyPdEndpoint
> [2021-09-23T11:10:17.426Z] 
> [2021-09-23T11:10:17.426Z] util_test.go:68:
> [2021-09-23T11:10:17.426Z]     c.Assert(verifyPdEndpoint(url, false), check.ErrorMatches, ".*invalid control character in URL.*")
> [2021-09-23T11:10:17.426Z] ... error string = "" +
> [2021-09-23T11:10:17.426Z] ...     "parse PD endpoint: parse \n" +
> [2021-09-23T11:10:17.426Z] ...     " hi: net/url: invalid control character in URL"
> [2021-09-23T11:10:17.426Z] ... regex string = ".*invalid control character in URL.*"
> [2021-09-23T11:10:17.426Z] 
> [2021-09-23T11:10:17.426Z] OOPS: 13 passed, 1 FAILED
> [2021-09-23T11:10:17.426Z] --- FAIL: TestSuite (0.06s)
> [2021-09-23T11:10:17.426Z] FAIL
> [2021-09-23T11:10:17.426Z] coverage: 33.6% of statements
> [2021-09-23T11:10:17.426Z] FAIL	github.com/pingcap/ticdc/cmd	0.216s
> ```

this error is weird...

**Pull request merged**